bin/kubeconfig.bash

```diff
@@ -93,7 +93,8 @@ kubectl --kubeconfig="${user_kubeconfig}" config set-credentials "${1}@${cluster
     --exec-arg=--oidc-issuer-url="https://dex.${base_domain}" \
     --exec-arg=--oidc-client-id=kubelogin \
     --exec-arg=--oidc-client-secret="$(sops -d --extract '["dex"]["kubeloginClientSecret"]' "${secrets[secrets_file]}")" \
-    --exec-arg=--oidc-extra-scope=email
+    --exec-arg=--oidc-extra-scope=email \
+    --exec-arg=--oidc-extra-scope=groups
```

Comment:
This is new right? Worth mentioning in changelog?

Reply:
I guess it's worth menitoning, this however will be completely transparent since if you don't have any groups from your IdP nothing will change and if you have groups you simply get it as expected.